src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+/**
+ * Ui deals with the user interactions where it outputs a message based on what
+ * which Command was given by the Parser.
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+    /**
+     * Produces a consistent line for everything the bot says.
+     *
+     * @return A line of hyphens.
+     */
+    public String lineProducer() {
+        String line = "   ------------------------------------------------------------------";
+        System.out.println(line);
+        return line;
+    }
+
+    /**
+     * Does a consistent indentation for the line where the bot outputs a message.
+     *
+     * @return An extra line and 4 spaces.
+     */
+    public String indentationAdder() {
+        return "\n    ";
+    }
+
+    /**
+     * The message that first greets the user when the program starts.
+     */
+    public void welcomeMessage() {
+        lineProducer();
+        System.out.println("    Hello I'm your friendly task-planning chatbot Duke!" +
+                indentationAdder() + "To enter a deadline, please type in this format: 'deadline {title of item} /by d/mm/yyyy hh:mm'" + indentationAdder() +
```

Comment:
Perhaps you can consider breaking this String up so that it does not exceed 120 characters?